src/sdg_hub/utils/__init__.py

```
@@ -3,3 +3,7 @@
 # This is part of the public API, and used by instructlab
 class GenerateException(Exception):
     """An exception raised during generate step."""
+
+from .path_resolution import resolve_path
```

Flake8 (7.2.0)

[error] expected 2 blank lines after class or function definition, found 1

(E305)